pkg/api/aim/apps.go

```diff
@@ -62,6 +86,7 @@ func GetApp(c *fiber.Ctx) error {
 	}
 	if err := database.DB.
 		Where("NOT is_archived").
+		Where("namespace_id = ?", ns.ID).
```

Comment:
move this into the `app` struct initialised above

Reply:
ahh. should be done.